gitenberg/book.py

```diff
@@ -282,3 +282,13 @@ def add_covers(self):
         return comment
 
 
+def list_all_books():
+    """Yields the full names of all the book repositories."""
+    if not config.data:
+        config.ConfigFile()
+
+    for repository in github3.login(
+            username=config.data['gh_user'],
+            password=config.data['gh_password']).organization(GITHUB_ORG).repositories():
```

Comment:
have you tested this command, i.e. does it use up 1 api transaction or 50,000 transactions?